docs/eventing/sources/pingsource.md

```diff
@@ -1,196 +1,261 @@
+kail -l serving.knative.dev/service=event-display -c user-container --since=10m
 ```
 
-This returns the `Attributes` and `Data` of the events that the PingSource sent to the `event-display` Service:
+For more information, see the [`kail`](https://github.com/boz/kail) repository in GitHub.
+
+{{< /tab >}}
+{{< /tabs >}}
+
+
+Log entries appear showing the request headers and body from the source:
 
 ```shell
 ☁️  cloudevents.Event
 Validation: valid
 Context Attributes,
   specversion: 1.0
   type: dev.knative.sources.ping
-  source: /apis/v1/namespaces/pingsource-example/pingsources/test-ping-source
-  id: 49f04fe2-7708-453d-ae0a-5fbaca9586a8
-  time: 2021-03-25T19:41:00.444508332Z
+  source: /apis/v1/namespaces/default/pingsources/test-ping-source
+  id: d8e761eb-30c7-49a3-a421-cd5895239f2d
+  time: 2019-12-04T14:24:00.000702251Z
   datacontenttype: application/json
 Data,
   {
     "message": "Hello world!"
   }
 ```
 
-If you created a PingSource with binary data, you should also see the following:
+If you created a PingSource with binary data, you also see the following:
 
 ```shell
 ☁️  cloudevents.Event
 Validation: valid
 Context Attributes,
   specversion: 1.0
   type: dev.knative.sources.ping
-  source: /apis/v1/namespaces/pingsource-example/pingsources/test-ping-source-binary
-  id: ddd7bad2-9b6a-42a7-8f9b-b64494a6ce43
-  time: 2021-03-25T19:38:00.455013472Z
+  source: /apis/v1/namespaces/default/pingsources/test-ping-source-binary
+  id: a195be33-ff65-49af-9045-0e0711d05e94
+  time: 2020-11-17T19:48:00.48334181Z
   datacontenttype: text/plain
 Data,
-  data
+  ZGF0YQ==
 ```
 
-### Cleanup
+## Clean up
+
+Delete the PingSource instance:
 
-Delete the `pingsource-example` namespace and all of its resources from your
-cluster by entering the following command:
+{{< tabs name="delete-source" default="kubectl" >}}
+{{% tab name="kubectl" %}}
+
+Delete the PingSource instance by running:
 
 ```shell
-kubectl delete namespace pingsource-example
+kubectl delete pingsources.sources.knative.dev test-ping-source
+kubectl delete pingsources.sources.knative.dev test-ping-source-binary
 ```
 
-## Reference Documentation
+{{< /tab >}}
 
-See the [PingSource specification](../../reference/api/eventing/#sources.knative.dev/v1beta2.PingSource).
+{{% tab name="kn" %}}
+
+Delete the PingSource instance by running:
+
+```shell
+kn source ping delete test-ping-source
+kn source ping delete test-ping-source-binary
+```
+
+{{< /tab >}}
+{{< /tabs >}}
 
-## Contact
 
-For any inquiries about this source, please reach out on to the
-[Knative users group](https://groups.google.com/forum/#!forum/knative-users).
+Delete the service instance:
+
+{{< tabs name="delete-service" default="kubectl" >}}
+{{% tab name="kubectl" %}}
+
+Delete the Service instance by running:
+
+```shell
+kubectl delete service.serving.knative.dev event-display
+```
+
+{{< /tab >}}
+
+{{% tab name="kn" %}}
+
+Delete the service instance by running:
```

Comment:
```suggestion
Delete the service instance:
```

Reply:
I think "by running" matters because it better orients readers, especially blind ones. I've written "Documenting Commands" rules in the Style Guide given Google's Style Guide is a little light on them.